### ISSUE TITLE
Swagger UI fails to execute request when one of query parameters is not filled out for the first time

### ISSUE THREAD

**Issue by pifchu** [springdoc/springdoc-openapi#1130]
**Describe the bug**
Swagger UI fails to execute request when one of query parameters is not filled out for the first time. It works only when you fill out all the query parameters (then you can remove any of the filled out parameters and no error would occur). This is happenning after 1.3.9 -> 1.5.6 SpringDoc version upgrade.
Stacktrace from the browser:

```
system.js:464 TypeError: Cannot read property 'toString' of undefined
    at fr (content-serializer.js:18)
    at dr (parameter-builders.js:45)
    at index.js:250
    at Array.forEach (<anonymous>)
    at Object.Or [as buildRequest] (index.js:205)
    at actions.js:452
    at utils.js:177
    at bindActionCreators.js:3
    at wrap-actions.js:33
    at Object.r (system.js:174)
```

**To Reproduce**
Spring Boot 2.4.4
SpringDoc 1.5.6
Example controller:
```java
@Operation(description = "Find notifications")
    @GetMapping(produces = MediaTypes.HAL_JSON_VALUE)
    @Parameter(name = "page", in = ParameterIn.QUERY, description = "Page number (0..n)",
            content = @Content(schema = @Schema(type = "integer", defaultValue = "0")))
    @Parameter(name = "size", in = ParameterIn.QUERY, description = "Page size",
            content = @Content(schema = @Schema(type = "integer", defaultValue = "20")))
    @Parameter(name = "sort", in = ParameterIn.QUERY, description = "Sorting criteria: name,asc|desc",
            content = @Content(array = @ArraySchema(schema = @Schema(type = "string"))))
    ResponseEntity<PagedModel<NotificationResponseModel>> find(
            @PageableDefault(size = 20, sort = "createdAt", direction = Sort.Direction.DESC)
            @Parameter(hidden = true) Pageable pageable);
```

1. Execute the given request after opening Swagger UI - error
2. Fill out any "sort" parameter - no errors
3. Remove "sort" parameter - no errors

Basically all our controllers that accept query parameters and return PagedModel have the same issue in Swagger UI.

**Expected behavior**
Requests executed as expected.

**Screenshots**
![image](https://user-images.githubusercontent.com/8541827/113666161-7d483d80-96b7-11eb-8bad-e1d6da4e8e0f.png)

**Comment by bnasslahsen** [comment]
@pifchu, 

 This JS error is related to the swagger-ui and not handled on the springdoc-openapi code.
 If you believe you are facing an issue, Please log your issue on the swagger-ui  github instead.
